test/functorch/test_aotdispatch.py

```diff
@@ -6184,6 +6185,68 @@ def fn(x):
         out_buffer = out.values()
         ga, gb, gc = torch.autograd.grad(out_buffer.sum(), (a, b, c))
 
+    def test_unwrap_async_collective_tensor_tangent(self):
+        class WrapSC(torch.Tensor):
```

Comment:
nit: it's starting to feel like we have a lot of one-off subclasses across our tests. Any reason not to just re-use `TwoTensor` here?

Reply:
> nit: it's starting to feel like we have a lot of one-off subclasses across our tests. Any reason not to just re-use `TwoTensor` here?

I was thinking to extract it to some utils module. Will try to use TwoTensor, I think it should work, I will try.

Reply:
TwoTensor works, removed redundant subclass in the test.